appIcons.js

```diff
@@ -166,7 +166,13 @@ const MyAppIcon = new Lang.Class({
 
     _updateRunningStyle: function() {
         this.parent();
```

Comment:
We hide the dots for apps that are not active in the current workspace.
Then we need to tell the AppSystem that changes were introduced.